JustSaying.IntegrationTests/WhenRegisteringASqsSubscriber/WhenHandlingMultipleTopics.cs

```diff
@@ -34,13 +34,13 @@ public async Task SqsPolicyWithAWildcardIsApplied()
 
             policyJson.Statement.Count.ShouldBe(1,  $"Expecting 1 statement in Sqs policy but found {policyJson.Statement.Count}");
         }
+    }
 
-        public class TopicA : Message
-        {
-        }
+    public class TopicA : Message
```

Comment:
Meh. I’d prefer to suppress this and keep them nested. Otherwise they have scope everywhere else in the test suite.